### PR TITLE
added Checkout page and razorpay integration

### PR DESCRIPTION
**Pull request by Rohanmond**
preview => https://fashi-fy-react-okppl6kvp-rohanmond.vercel.app/

### CODE REVIEW

File: fashify/src/App.js [hunk 1]
```diff
@@ -13,7 +17,7 @@ import Login from './pages/Auth/Login/Login';
 import { useAuth } from './contexts/auth-context';
 import Loader from './components/Loader/Loader';
 import { useData } from './contexts/data-context';
```
Comment (znjs) [inline fashify/src/App.js:19]: can import from context directly since you have defined `index.js` in context

**Review by anuj-kumary — APPROVED** [review]

File: fashify/src/Services/services.js [hunk 2]
```diff
@@ -78,6 +80,7 @@ export const IncDecCart = async ({ productId, encodedToken, type }) => {
   );
 };
 export const DeleteCart = async ({ productId, encodedToken }) => {
+  console.log('inservei delete');
```
Comment (znjs) [inline fashify/src/Services/services.js]: can remove console

**Review by PeeyushPrashant — APPROVED** [review]
looks good to merge

File: fashify/src/pages/Checkout/Checkout.jsx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+      'https://checkout.razorpay.com/v1/checkout.js'
+    );
+
+    if (!res) {
+      toast.error('Razorpay SDK failed to load, check you connection');
+      return;
+    }
+
+    const options = {
+      key: 'rzp_test_SR2urKhQGjFxHb',
```
Comment (znjs) [inline fashify/src/pages/Checkout/Checkout.jsx:103]: can add this key to `.env` file and use a variable here

**Review by znjs — APPROVED** [review]
- Good folder structure
- Good naming conventions used
- feature working as expected

**Pull request merged**